flexmeasures/api/common/schemas/assets.py

```diff
@@ -28,10 +51,27 @@ class AssetAPIQuerySchema(PaginationSchema):
             example=2,
         ),
     )
+    included_fields = PipedListField(
+        load_default=None,
```

Comment:
Why not put the default here already as the `load_default` (so Swagger might pick it up, too)?

Reply:
Yes, good catch